src/RPtest.sch

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<sch:schema xmlns:sch="http://purl.oclc.org/dsdl/schematron" queryBinding="xslt2"
```

Comment:
Suggest changing `xmlns:sch="http://purl.oclc.org/dsdl/schematron"` to `xmlns="http://purl.oclc.org/dsdl/schematron"`.

We can then get rid of the `sch:` namespace at the start of elements, i.e. `<schema ...`